source/Government.cpp

```diff
@@ -216,6 +218,25 @@ void Government::Load(const DataNode &node)
 			enforcementZones.emplace_back(child);
 		else if(key == "provoked on scan")
 			provokedOnScan = true;
+		else if((child.Token(0) == "interdiction" || child.Token(0) == "interdiction bribe")
+				&& child.Size() >= 2)
```

Comment:
Use key instead of child.tokem(0) and move this after the value check of line 243 so we dont have to check child.size